test/test_ops.py

```diff
@@ -215,8 +215,7 @@ def test_variant_consistency_eager(self, device, dtype, op):
             for variant in variants:
                 # Verifies that inplace operations that promote int->float fail
                 #   on tensors with integer dtypes.
-                if (variant is inplace and op.promotes_integers_to_float and
-                        dtype in (torch.bool, torch.uint8, torch.int8, torch.int16, torch.int32, torch.int64)):
+                if (variant is inplace and not torch._C._can_cast(expected_forward.dtype, dtype)):
```

Comment:
We can just call the torch.can_cast here, right?

https://pytorch.org/docs/master/generated/torch.can_cast.html?highlight=can_cast#torch.can_cast

Reply:
Ah, I hadn't realised it was already exported to python.